ug-vmware-users/charter.md

```diff
@@ -0,0 +1,43 @@
+# VMware User Group Charter
+
+This User Group adheres to the governance described in [ug-governance] under the community standards of the Kubernetes project.
+
+## Scope
+The VMware User Group facilitates communication among Kubernetes users and contributors on topics pertaining to running all [conformant] forms of Kubernetes on VMware infrastructure. Scope includes extensions and integration of Kubernetes with Kubernetes subprojects.
+
+In addition to the usage of Kubernetes running on the cloud provider for the vSphere hypervisor, the User Group’s mission includes:
```

Comment:
I'm confused where the line is here between the [provider-vsphere subproject](https://github.com/kubernetes/community/tree/master/sig-cloud-provider#provider-vsphere) under SIG Cloud Provider and this user group

Reply:
+1, some elaboration on this in the charter would be great!

Two points mentioned here are also mentioned in the (old) SIG VMware's charter: https://github.com/kubernetes/community/blob/master/sig-vmware/charter.md#scope